app/models/health_care_application.rb

```diff
@@ -71,7 +71,7 @@ def send_failure_email?
   end
 
   def submit_sync
-    @parsed_form = override_parsed_form(parsed_form)
+    @parsed_form = HCA::OverridesParser.new(parsed_form).override
```

Comment:
I think you will need to require the HCA::OverridesParser file here. `/lib` is not autoloaded

Reply:
Added! I also added it to `lib/form1010_ezr/service.rb`. Thank you!